zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/job/ActivateJobsHandler.java

```diff
@@ -7,37 +7,24 @@
  */
 package io.camunda.zeebe.gateway.impl.job;
 
-import io.camunda.zeebe.gateway.grpc.ServerStreamObserver;
 import io.camunda.zeebe.gateway.impl.broker.request.BrokerActivateJobsRequest;
-import io.camunda.zeebe.gateway.protocol.GatewayOuterClass.ActivateJobsResponse;
 import io.camunda.zeebe.scheduler.ActorControl;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.Consumer;
 
 /** Can handle an 'activate jobs' request from a client. */
-public interface ActivateJobsHandler extends Consumer<ActorControl> {
+public interface ActivateJobsHandler<T> extends Consumer<ActorControl> {
 
   static final AtomicLong ACTIVATE_JOBS_REQUEST_ID_GENERATOR = new AtomicLong(1);
 
   /**
    * Handle activate jobs request from a client
    *
    * @param request The request to handle
-   * @param responseObserver The stream to write the responses to
    */
   void activateJobs(
-      BrokerActivateJobsRequest request,
-      ServerStreamObserver<ActivateJobsResponse> responseObserver,
-      final long requestTimeout);
-
-  public static InflightActivateJobsRequest toInflightActivateJobsRequest(
```

Comment:
ℹ️ This method was never used internally and the class is not public API (as far as I can tell). Thus, I think removing this method is fine.